docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,70 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+### Instructions
+
+* (1) [Fork the REANA main repository](https://github.com/reanahub/reana/fork) and clone it:
+
+```console
+$ git clone https://github.com/johndoe/reana.git
```

Comment:
`git clone git@github.com:johndoe/reana` since we use `git` protocol in other git commands, not `https` protocol.